core/cmd/local_client_test.go

```diff
@@ -51,7 +52,7 @@ func TestClient_RunNodeShowsEnv(t *testing.T) {
 
 	db := pgtest.NewSqlxDB(t)
 	pCfg := cltest.NewTestGeneralConfig(t)
-	sessionORM := sessions.NewORM(db, time.Minute, lggr, pCfg)
+	sessionORM := sessions.NewORM(db, time.Minute, lggr, pCfg, &audit.AuditLoggerService{})
```

Comment:
A couple of loose `&audit.AuditLoggerService{}` in here that could point to `audit.NoopLogger` instead.